### ISSUE TITLE
key_file mount option results in "fusermount: mount failed: Invalid argument"

### ISSUE THREAD

**Issue by craigafinch** [GoogleCloudPlatform/gcsfuse#183]
I am able to mount a bucket using 
`gcsfuse --key-file=/keyfile.json bucket_name /my/mount/point`

or 

` GOOGLE_APPLICATION_CREDENTIALS=/keyfile.json gcsfuse bucket_name /my/mount/point`

I added a line to `fstab` as follows:

`bucket_name  /my/mount/point gcsfuse rw,auto,user,key_file=/keyfile.json`

```
mount /my/mount/point 

Calling gcsfuse with arguments: -o nosuid -o nodev -o key_file=/keyfile.json -o rw -o noexec stage-user_files /my/mount/point
Using mount point: /my/mount/point
Opening GCS connection...
Opening bucket...
Mounting file system...
daemonize.Run: readFromProcess: sub-process: mountWithArgs: mountWithConn: Mount: mount: running fusermount: exit status 1

stderr:
fusermount: mount failed: Invalid argument

running gcsfuse: exit status 1
```

I also tried using `key-file` in place of `key_file`, which produced the same result.

**Comment by jacobsa** [comment]
This is likely because 81281027c0093e3f916a6e611a128ec5c3a12ece hasn't yet made it into a release. I will try to do a release soon.

**Comment by craigafinch** [comment]
Thank you! Let me know if you would like me to test a release candidate.

**Comment by jacobsa** [comment]
Should be fixed in 0.19.0. Let me know if not.

**Comment by craigafinch** [comment]
I confirm that this issue is resolved in 0.19.0 on CentOS 7 (Google Compute Engine).
